project.clj

```diff
@@ -136,7 +136,7 @@
                          [org.bouncycastle/bcpkix-fips "1.0.3"]
                          [org.bouncycastle/bc-fips "1.0.2"]
                          [org.bouncycastle/bctls-fips "1.0.10"]
-                         [org.bouncycastle/bcpkix-jdk15on "1.60"]]
+                         [org.bouncycastle/bcpkix-jdk15on "1.65"]]
```

Comment:
I think latest is 1.66 now

Reply:
Oh. Hm. Might need to re-test then.

Reply:
`ssl-utils` was green, doing server now.

Reply:
Server is also green.